data-api/the-metrics/index.rst

```diff
@@ -0,0 +1,86 @@
+.. _metrics:
+
+The metrics
+===========
+
+1. Passing By
+-------------
+
+  * **Keyword**: ``passing_by``
+  * **Schema**:
+
+    .. code:: json
+
+      { "value": 0 }
+
+  * **Description**: Count of people who passed (not necessarily stopping or
+    looking) in front of the camera during the specified time duration.
```

Comment:
It is 1s

Reply:
Count of people who passed (not necessarily stopping or looking) in front of the camera during at least 1 second.